src/core/presence.ts

```diff
@@ -55,6 +81,11 @@ export interface PresenceEvent {
    */
   clientId: string;
 
+  /**
+   * The connectionId of the presence member.
+   */
+  connectionId: string;
```

Comment:
Should we provide some method here to help users with determining unique members in the presence set, e.g. `memberKey()`?